nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -12,6 +12,8 @@
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
+- [Melange](https://melange.re/), the toolchain to produce JavaScript from Reason/OCaml.
```

Comment:
for now this is just a package, so I would leave the release note out

Reply:
Ok, I just removed that entry from my commit.